tests/network.rs

```diff
@@ -1,8 +0,0 @@
-extern crate sfml;
```

Comment:
This never really made sense as an integration test anyway, So I moved it into a unit test inside the network module.